setup.py

```diff
@@ -70,6 +71,6 @@ def run_tests(self):
         'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
         ],
     extras_require={
-        'testing': ['pytest'],
+        'testing': ['pytest', 'tox', 'flake8', 'coverage'],
```

Comment:
@rconradharris Not sure if this is strictly required given that tox seems to be able to install stuff. I personally find it clearer to have the testing dependencies listed here. I found that after running tox, I still didn't have flake8 installed.